src/main/webapp/script.js

```diff
@@ -130,6 +158,7 @@ async function generateGraph() {
 
   mutationIndexList = JSON.parse(jsonResponse.mutationIndices);
   numMutations = mutationIndexList.length;
+  maxNumMutations = jsonResponse.totalMutNumber;
```

Comment:
is there any reason you created a setter but didn't use it?

Reply:
I used it in the test file. I don't know if it makes a lot of sense to use a setter in the same file as the variable. Do you think I should do that?